### PR TITLE
HTMLBook build fixes

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

**Comment by jni** [comment]
@stefanv is this ready to merge?

File: markdown/ch1.markdown [hunk 1]
```diff
@@ -797,32 +798,36 @@ and the gene length.
...
+Let's say:
+
+C = Number of reads mapped to a gene
+L = exon length in base-pairs for a gene
+N = Total mapped reads in the experiment
 
 First, let's calculate reads per kilobase.
 
-Reads per base would be:  
+Reads per base would be:
```
Comment (jni) [inline markdown/ch1.markdown]: @stefanv did you decide this looks better inline?

File: markdown/ch1.markdown [hunk 2]
```diff
@@ -797,32 +798,36 @@ and the gene length.
...
 
 But biologists like thinking in millions of reads so that the numbers don't get
-too big. Counting per million reads we get:  
+too big. Counting per million reads we get:
+
 $ \frac{10^3C}{L(N/10^6)} = \frac{10^9C}{LN}$
 
 
-In summary, to calculate reads per kilobase transcript per million reads:  
+In summary, to calculate reads per kilobase transcript per million reads:
```
Comment (jni) [inline markdown/ch1.markdown]: Same comment as above...

File: markdown/ch1.markdown [hunk 3]
```diff
@@ -859,22 +864,26 @@ Once the two arrays have the same number of dimensions,
 broadcasting can only occur if the sizes of the dimensions match,
 or one of them is equal to 1.
 
-For example, let's say we have two ndarrays, A and B:  
-`A.shape = (1, 2)`  
+For example, let's say we have two ndarrays, A and B:
+`A.shape = (1, 2)`
```
Comment (jni) [inline markdown/ch1.markdown]: These will now all render inline?

File: markdown/ch3.markdown [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Networks of Image Regions with ndimage
 
-**Code by Vighnesh Birodkar.**  
+**Code by Vighnesh Birodkar.**
```
Comment (jni) [inline markdown/ch3.markdown]: Same as above?

File: markdown/ch4.markdown [hunk 5]
```diff
@@ -1018,22 +1020,22 @@ a function $x(t)$ of time (or another variable, depending on the
 application) into a sequence of $N$ complex numbers $X_{k}$ by the
 summation
 
-$$X_{k}=\sum_{n=0}^{N-1}x_{n}e^{-j2\pi kn/N},\;k=0,1,\ldots
-N-1.\label{eq:Forward DFT}$$
+$X_{k}=\sum_{n=0}^{N-1}x_{n}e^{-j2\pi kn/N},\;k=0,1,\ldots
+N-1.\label{eq:Forward DFT}$
```
Comment (jni) [inline markdown/ch4.markdown]: Uh oh — does HTMLBook not allow equation mode?

**Comment by stefanv** [comment]
The changes here allow HTMLBook.js to produce valid HTMLBook.  Removing trailing spaces should not matter, I think, but the equation mode is annoying.  We may want to add a custom parsing step for that.  For now, these are the minimal changes needed to compile the book; admittedly, I have not had a look at the notebook output.

**Comment by jni** [comment]
@stefanv were your ATLAS builds based on this PR, or other changes?

**Comment by stefanv** [comment]
Based on this PR, but without Harriet's latest edits.  I will make an
upload of those.




On Thu, Jan 19, 2017, at 21:00, Juan Nunez-Iglesias wrote:

> @stefanv[1] were your ATLAS builds based on this PR, or other changes?
> — You are receiving this because you were mentioned. Reply to this
> email directly, view it on GitHub[2], or mute the thread[3].
> 




Links:

  1. https://github.com/stefanv
  2. https://github.com/elegant-scipy/elegant-scipy/pull/220#issuecomment-273978071

**Comment by stefanv** [comment]
Done for now, please merge.

**Comment by stefanv** [comment]
One massive leap for mankind!

**Pull request merged**